.ci/ci.R

```diff
@@ -169,7 +169,8 @@ function(pkgs,
     newpkgs <- newpkgs[availpkgs]
   }
 
-  pkgsext <- switch(type,
+  typeshort <- if (substr(type, 1L, 11L) == "mac.binary.") "mac.binary" else type
```

Comment:
```suggestion
  typeshort <- if (startsWith(type, "mac.binary.")) "mac.binary" else type
```

Or `gsub("(mac.binary)[.].*", "\\1", type)`

Reply:
Corresponding line is taken from `contrib.url(repos, type)` which does the same for `type`